src/atoms/gv-autocomplete.js

```diff
@@ -270,11 +274,17 @@ export class GvAutocomplete extends LitElement {
       options: true,
       open,
     };
+
+    const noOptionClasses = {
+      shouldShowNoOption,
+    };
     return html` <div class="container">
       ${this._renderStyle()}
       <slot></slot>
-      <div class="${classMap(classes)}" @mouseleave="${this._onMouseLeave}" style="${styleMap({ top: `${top}px` })}">
+      <div class="${classMap(noOptionClasses)}" style="${styleMap({ top: `${top}px` })}">
```

Comment:
Where is mouseleave binding ?

Reply:
It is at L286, on the options div. This is a new div that contains only `noOption` slot.